salesagility/SuiteCRM
# Fix #719 YUI https - Blocked loading mixed active content

## Description
This changes any external YUI script loading to use https, to prevent mixed content warning and scripts being blocked

## How To Test This
Rebuild cache js, clear browser cache and see that any external YUI scripts use https, e.g in calendar 

## Types of changes


### Final checklist
